vignettes/estimate_infections_workflow.Rmd

```diff
@@ -91,17 +91,27 @@ Gamma(mean = 3, sd = 1, max = 10)
 #>     3
 ```
 
+which looks like this when plotted
+
+
+``` r
+plot(Gamma(mean = 3, sd = 1, max = 10))
+```
+
+![plot of chunk unnamed-chunk-9](estimate_infections_workflow-unnamed-chunk-9-1.png)
```

Comment:
I'd suggest to give the chunks meaningful names so that the files have them too

Reply:
See 0cda4404a6a3489f10fee7a25a87139747f8c86b